---
title: "Developing novel methods for gene editing in trees"
header-includes:
- \usepackage{float}
- \usepackage{booktabs}
- \usepackage{fancyhdr}
- \pagestyle{fancyplain} % use fancy for all pages except chapter start
- \lhead{\includegraphics[height=1.2cm]{telhai logo.jpg}} % left logo
- \rhead{\includegraphics[height=1.2cm]{MigalLogo.png}} % right logo
- \renewcommand{\headrulewidth}{0pt} % remove rule below header
- \cfoot{\thepage}
author: "Ben Sivan"
date: "april 2021"
output: 
   pdf_document:
     number_sections: yes
always_allow_html: true
bibliography: library.bib
csl: Taylor & Francis - Council of Science Editors.csl
---

\pagenumbering{gobble}

\begin{center}
   \textbf{\large Under the supervision of\\
                      Dr. Amir Raz\\
                  Prof. Martin Goldway}
\end{center}

\vspace{\fill}

Signature of student:_______________Date:____/____/_______ \hfill \break \newline Signature of supervisor:_______________Date:____/____/_______\hfill \break \newline Signature of supervisor:_______________Date:____/____/_______ \hfill \break \newline Signature of chairperson for graduate studies:_______________Date:____/____/_______

\begin{titlepage}
   \begin{center}
       
        \includegraphics[width=0.5\textwidth]{telhai logo.jpg}
        
        \vspace{0.5cm}

       \textbf{\Huge THE FACULTY OF SCIENCES\\ MASTER IN BIOTECHNOLOGY}

       \vspace{2cm}
       \textbf{\huge Developing novel methods for gene editing in trees}
            
       \vspace{3cm}

       \textbf{\Large Ben Sivan}
       
        \vspace{1cm}

       \textbf{\large Under the supervision of\\
       Dr. Amir Raz\\
       Prof. Martin Goldway}
       
       \vspace{1cm}

       \textbf{\large Thesis submitted in partial fulfillment of the requirements for the master of science degree in biotechnology}
       
       \vfill
     
       Department of Biotechnology\\
       Tel-Hai collage\\
       Israel\\
       October 2021
            
   \end{center}
\end{titlepage}

\newpage

\setcounter{tocdepth}{3}
\tableofcontents
\newpage

\listoffigures
\newpage

\listoftables
\newpage

\pagenumbering{arabic} 
\setcounter{page}{7}

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE, fig.pos= "H")
```

```{r library, echo=FALSE, message=FALSE, warning=FALSE, results='asis'}
library(knitr)
library(kableExtra)
library(tidyverse)
library(ggpubr)
library(latex2exp)
```

# Abstract

Global environmental change undermines food security. In comparison to most crops, trees are resilient to temperature fluctuations and
consequently offer vital insurance against famine. Crop improvement with available methods reaches a glass ceiling and genetic modification can contribute substantially to break through the barrier. Tissue culture is a required step in most genetic modification methods. Yet, tissue culture has its issues and the process is far from routine in most laboratories. In trees, because of long generation time, tissue culture issues become much more pronounced. In this work we are attempting to implement a novel gene editing method that doesn't require tissue culture such as de-novo meristem induction and transformation, on trees. In this work we used _Agrobacterium tumefaciens_ as the vector for in-planta transformation. We used apples as a commercial fruit tree crop and poplar as a model system for trees. We experimented with a wide range of development regulators, and a few agroinfiltration strategis. Additionally we analyse tissue specific transcriptome to further understand gene expression patterns in tissue development, and scan for better normalizer genes for real-time quantitive PCR. In this work We have shown that over expression of the gene combination _WUS2-STM_ and _WOX11-STM_ were successful in invoking de-novo shoot regeneration in a young apple plants, in-planta. Further we found ,based on the tissue specific transcriptome, that in poplar the most shoot tip specific transcription factor from the WUSCHEL family is WOX1. 

# Introduction

## Trees and their vital role as source for food security

Food security is a fundamental necessity that kept mankind busy from the beginning of time. Many researchers today estimate that this need is under threat as a result of global environmental changes through land degradation, loss of biodiversity, changes in hydrology, and changes in climate patterns[@Ericksen2009]. Moreover population growth is around 100 milion per year. Since the early 1990s, the number of extreme weather-related disasters has doubled[@FAO2020a]. Higher temperatures, water scarcity, extreme events like droughts and floods have already begun to impact staple crops around the world[@Linden2015] and have reduced the yields of major crops like maize and wheat[@Rosenzweig2001]. According to the Food and Agriculture Organization of the United Nations, the climate variability has an impact of at least 80% to the unpredictable reduction in yield of cereal crops in semi-arid regions of the world such as the Sahel region of Africa[@Shiferaw2014].

Fruit trees contribute in many ways to improving diets and combating hunger around the world[@Vinceti2013]. Trees are much more resilient to extreme weather-related disasters in comparison to most crops, and consequently they can offer vital insurance against famine during times of seasonal food shortages due to droughts, floods and heat/cold waves. This is the main reason behind the evergreen agriculture approach[@Garrity2010]. Trees resilient is due to their being perennial woody plants, which allows them to grow strong, with durable trunk and long roots. In addition, fruit trees are able to produce large yield on a given area resulting from  their vertical growth. However, plant improvement methods raises a drawback since trees have a long generation time.

## Classical plant improvement methods for trees on a human time scale

Since mankind developed the ability to grow plants for food, methods were developed to improve yield. The most ancient method is the selection and propagation of plants with preferred traits such as larger seeds. This method creates an artificial selection pressure that allows the highlighting of desirable traits in the plant. However this process is dependent on random mutations, making it slow. Further more, undesirable additional traits may appear.

The second method that man has developed for improving the crop is breeding. Breeding at its core is based on the idea of merging traits from differente lines to a single cultivar through crossbreeding. In most cases, there is an all ready main cultivar line that harbors a variety of desirable triats and probably used as a commertial line. That cultivar would be crossbred with some exotic line whose over all traits (or phynotype) are not at the commercial standart but withhold some favorable trait. The descendants of that cross would have equal parts of genetic matirial donated from each of the parents, and through the proccess of backcrossing with the commertial parent and selecting for offsprings based on thier phynotype, it is possible to narrow in on a descendant with much of the commercial phynotype but that also keeps the desirable trait from the exotic line. This is why the commercial parent is often called the recurrent parent, and the exotic one is called the donor. Although this method is faster, this proccess takes many generations and even in annual plants takes a few years. Still, impressive results have been obtained in annual crops that have changed the fate of the human race to extremes. In trees, the time from polination to a grown fruit bearing tree takes a few years in of itself. As a result, to take the breeding proccess as is and apply it on trees takes decades. Therefor the realization of the potential in trees is not as high.

## Genetic modification as a viable solution for trees improvement

With the understanding that the inherited genetic material is DNA, the ability was developed to utilize the acceleration of random mutations as a way of gaining genetic diversity and selecting based on the phynotipic outcome. By exposing seeds to chemicals or radiation we are able to increase the frequency of mutation events, some fraction of those mutations results with desirable traits. This process is called mutation breeding and plants created using mutagenesis are sometimes called mutagenic plants or mutagenic seeds. There are different kinds of mutagenic breeding, for instance such as chemical mutagens like ethyl-methanesulfonate and dimethyl-sulfate, or radiation such as gamma rays and X-rays[@Schouten2007]. Although this method increase the rate of random mutation formation, its still relies on random occurrences and offspring selection.

As a result of the genetic revolution, many genetic modification technologies were developed, some of them relevant for plants too. Those methods are more direct in their approach to manipulate DNA for an expected outcome. They can be divided into two groups, exogenous DNA delivery methods and site-specific endonucleases. 

## Exogenous DNA delivery methods  
  
Techniques developed for the introduction of foreign DNA genes (exogenous) into a cell.

### Biolistic particle delivery system or Gene gun  

Gene gun or biolistic particle delivery system is a device used to deliver exogenous DNA (transgenes), RNA, or protein to cells. By coating particles of a heavy metal with a gene of interest and firing these micro-projectiles into cells using mechanical force, an integration of desired genetic information can be induced into cells. The technique involved with such micro-projectile delivery of DNA is often referred to as biolistics[@OBrien2011]. This device is able to transform almost any type of cell and is not limited to the transformation of the nucleus, it can also transform organelles, including plastids and mitochondria[@Rakoczy-Trojanowska2002]. Biolistics has proven to be a versatile method of genetic modification and it is generally preferred to engineer transformation-resistant crops, such as cereals. Notably, Bt maize is a product of biolistics. Biolistics introduces DNA randomly into the target cells. Thus the DNA may be transformed into whatever genomes are present in the cell, either nuclear, mitochondrial, plasmid or any others, in any combination, though proper construct design may mitigate this. The delivery and integration of multiple templates of the DNA construct is a distinct possibility, resulting in potential variable expression levels and copy numbers of the inserted gene[@Shewry2008149].

### Protoplast transformation  
  
Protoplast refers to the entire cell excluding the cell wall. Protoplasts can be generated by stripping the cell wall from plant, bacterial, or fungal cells by mechanical, chemical or enzymatic means[@davey2005plant]. The advantages of using protoplast in comparison to a whole cell is the direct access to the cells membrane which result in increase uptake of DNA and increase transformation frequency. Treatment of protoplast-plasmid mixtures with PEG and/or electroporation is the approach normally exploited to induce DNA into protoplasts. However, transformation frequencies typically remain low (ca. one in 104 protoplasts giving stably transformed tissues)[@davey2005plant]. Heat shock treatment and irradiation of recipient protoplasts enhance transformation frequency, probably by increasing the recombination of genomic DNA with incoming foreign DNA, or the initiation of repair mechanisms that favour DNA integration. Carrier DNA and the nature of the plant genome also affect transformation[@davey2005plant]. DNA uptake into protoplasts has been especially important in transforming plants that are not amenable to other methods of gene delivery, particularly agrobacterium-mediated transformation. Many of such studies focused on cereals, particularly rice, once protoplast-to-plant systems became available for these crops[@Rakoczy-Trojanowska2002]. However, protoplast regeneration into mature plants is hard to achieve in most plants, which is the major holdback of this approach.

### Agrobacterium-mediated transformation  
  
Agrobacterium is a genus of Gram-negative bacteria that uses horizontal gene transfer to cause tumors in plants. _Agrobacterium tumefaciens_ is the most commonly studied species in this genus. Agrobacterium is well known for its ability to transfer DNA between itself and plants, and for this reason it has become an important tool for genetic engineering. The ability of Agrobacterium to transfer genes to plants and fungi is used in biotechnology, in particular, genetic engineering for plant improvement. Genomes of plants can be engineered by use of Agrobacterium for the delivery of sequences hosted in transfer of a DNA segment (T-DNA) binary vector. The essential parts of the T-DNA are its two small (25 base pair) border repeats, at least one of which is needed for plant transformation. The genes to be introduced into the plant are cloned into a plant binary vector that contains the T-DNA region, together with a selectable marker (such as antibiotic resistance) to enable selection for plants that have been successfully transformed. Plants are grown on media containing antibiotics following transformation, and those that do not have the T-DNA integrated into their genome will die[@Mukeshimana2013]. The most common methodology for introducing Agrobacterium to plant tissues is in liquid suspension of sectors of somatic tissue (explant), then co-culture on agar medium in the dark. Another method is Agroinfiltration, used to induce transient expression of genes in a plant. Agroinfiltration is performed by direct injection or by vacuum infiltration of suspended _Agrobacterium tumefaciens_ into a plant leaf. The main benefit of agroinfiltration when compared to the more traditional plant transformation is speed and convenience, although yields of the recombinant protein in traditional methods are generally higher and more consistent[@Undervisningsministeriet2014]. Floral dipping is another method that allows efficient plant agrobacterium-mediated transformation without need for tissue culture[@Zhang2006]. 

### Virus as a DNA delivery vector

Virus mediated gene delivery utilizes the ability of a virus to inject their genetic matirial into a host cell and takes advantage of the virus’s ability to hijack the cells machinary for replication and integration to the genome. This method has been used to deliver Small interfering RNAs (siRNAs) in order to silence a certain gene and hence it is called Virus-Induced Gene Silencing.

### Virus-Induced Gene Silencing (VIGS)

Virus-induced gene silencing is one of the reverse genetics tools for analysis of gene function that uses viral vectors carrying a target gene fragment to roduce double-strand RNA (dsRNA) which trigger RNAmediated gene silencing. Small interfering RNAs (siRNAs) which are 21–25 nucleotide long, guides specific cleavage or suppression of target messenger RNA (mRNA) at posttranscriptional level. The process takes place as follows, siRNAs which are processed from long double-stranded RNAs (dsRNA) by DICER, an RNAse-like enzyme, are then incorporated into RNA-induced silencing complex (RISC). This complex with siRNA targets specific mRNA transcripts having sequence complementarity with the specific siRNA. In other words, the antisense strand of the siRNA associates with the RNAi silencing complex (RISC) to target homologous RNA for degradation. Some virus species were previously modified and used for silencing the gene of interest[@Jiang2014]. Tobacco mosaic virus (TMV) is the first modified virus for application of VIGS methods to plants. TMV was used for effective pds gene silencing in Nicotiana benthamiana plants[@Harries2008]. Tobacco rattle virus (TRV) was also modified to be a tool for gene silencing in plants. TRV has a bipartite genome, consisting of two positive-sense single-stranded RNAs, designated RNA1 and RNA2. The significant advantage of TRV-based VIGS in Solanaceous species is the ease of introduction of the VIGS vector into plants. The VIGS vector is placed between Rigth Border (RB) and Left Border (LB) sites of T-DNA and inserted into Agrobacterium tumefaciens. Another property of TRV is the more vigorous spreading all over the entire plant including meristem, and infection symptoms of TRV are mild[@Ratcliff2001]. Lately barley stripe mosaic virus (BSMV) was developed for efficient silencing of pds gene in barley. This system was then used for silencing of wheat genes. BSMV is a positive sense RNA virus containing a tripartite ($\alpha$,$\beta$,$\gamma$) genome. The modified $\gamma$ of BSMV genome allows simple cloning of silencing DNA fragment from the gene of interest. $\beta$ genome has been deleted for viral coat protein production defect[@fire1998potent].

## Site-specific endonucleases

Site-specific endonucleases are enzymes that are capable of dissecting nucleic acid strands such as DNA or RNA at a specific target sequence. In contrast to restriction enzymes, this sequence can be engineered and hold longer sequences which increase it's specificity.

### Zinc Finger Nucleases (ZFNs)   
  
A zinc finger is a small protein structural motif that is characterized by the coordination of one or more zinc ions (Zn2+) in order to stabilize the fold. The ability to engineer zinc fingers to have an affinity for a specific sequence of DNA, made them suitable for important applications such as zinc finger nucleases and zinc finger transcription factors. While significant progress has been made in ZFNs engineering capability, a barrier to their widespread adoption has been the challenge in engineering new DNA binding specificities[@Chandrasegaran1996].

### Transcription Activator-like Effector Nucleases (TALENs)  
  
Transcription activator-like effector nucleases are restriction enzymes that can be engineered to cut specific sequences of DNA. They are made by fusing a TAL effector DNA-binding domain to a DNA cleavage domain (a nuclease which cuts DNA strands). The simple relationship between amino acid sequence and DNA recognition of the TALE binding domain allows for the efficient engineering of proteins. That said, the failure of some custom TALENs suggests that yet unknown rules govern the assembly of functional repeat domains. For example, repeat composition may influence protein stability[@Christian2010].

### Clustered Regularly Interspaced Short Palindromic Repeat (CRISPR) 
  
CRISPR systems are part of the adaptive immune system of bacteria and archaea, protecting them from invading viruses by cleaving their DNA in a sequence-dependent manner. The immunity is acquired by the integration of short fragments of the invading DNA known as spacers between two adjacent repeats at the proximal end of a CRISPR locus. The CRISPR arrays, including the spacers, are transcribed during subsequent encounters with invasive DNA and are processed into small interfering CRISPR RNAs (crRNAs) approximately 40 nt in length, which combine with the transactivating CRISPR RNA (tracrRNA) to activate and guide the CRISPR Assosiated 9 (Cas9) nuclease[@Barrangou1709]. A prerequisite for cleavage is the presence of a conserved protospacer-adjacent motif (PAM) downstream of the target DNA, which usually has the sequence 5'-NGG-3'[@GasiunasE2579]. Jinek et al., re-engineered the Cas9 endonuclease into a more manageable two-component system by fusing the two RNA molecules into a "single-guide RNA"(sgRNA) that, when combined with Cas9, could find and cut the DNA target specified by the guide RNA. By manipulating the nucleotide sequence of the guide RNA, the artificial Cas9 system could be programmed to target any DNA sequence for cleavage[@Jinek2012]. The Nobel Prize in Chemistry 2020 was awarded to Emmanuelle Charpentier and Jennifer A. Doudna for the development of the CRISPR/Cas9 enzyme as a genetic editing tool.

## Genetic Editing bottleneck

Bottlenecks need to be overcome before the full potential of this technology is realized in plants. Plants distinguish themselves from most complex eukaryotes in the totipotency of their tissues[@Indra1972]. This has long allowed researchers to convert explant into a whole plants, with the use aseptic tissue culture growth medium. This somatic-germinal conversion (or regeneration) is the foundation of most plant transformation approaches. Transgenes are delivered to isolated somatic tissue followed by selection for the transgene and regeneration of the modified tissue into a whole, transgenic plant[@Rasmussen2017]. Despite many of these protocols being developed over decades, the process is far from routine in most laboratories. Further,success is often genotype dependent because specific growth medium need to be tuned for each new plant and the regenerated plants can have changes to their genome and epigenome[@Kaeppler2000]. In trees, because of long generation time, the issues of maintaining asceptic enviroment and assessing growth medium suitability in real time become much more pronounced.

Crop production globally is improving, but this trend seems to approach a plateau (Figure \ref{fig:agricultural area per capita}). Hence substantial changes in methods for agronomic processes and crop improvement are required[@TesterMarkandLangridge2010]. 

```{r agricultural area per capita, echo=FALSE, message=FALSE, warning=FALSE, fig.pos="H", out.width = "70%", fig.align="center", fig.cap="Agricultural area per capita world-wide in hectare/person. \\newline{} \\textbf{Line.} Average area \\textbf{Ribbon.} Maximum and minimum values  \\newline{} https://ourworldindata.org/"}

agricultural_area_per_capita_summary <- read.csv("Tables/agricultural_area_per_capita_summary.csv")

agricultural_area_per_capita_summary_fig = agricultural_area_per_capita_summary %>% ggplot(aes(x = Year)) + geom_ribbon(aes(ymin=Min, ymax=Max), fill = "grey70") + geom_line(aes(y = Avg)) + ylab("hactare/person") + theme_classic()

agricultural_area_per_capita_summary_fig
```


## Advance genetic Editing technologies

Most of the advanced methods of genetic editing are designed to help solve problems arising from the use of tissue culture, or to completely spare the need to use it. Tissue culture is designed to allow for the capture of rare cell-transformed events. This is done by providing a supportive environment for the explant, whose needed to expose as much as possible inner cells of the plant to the genetic manipulation and later allow for regeneration and differentiation to a new plant. For the most part, the regeneration begins with differentiation to a shoot meristem, and later to a full shoot. The route from shoot to a full plant through the induction of roots is much more approachable. The common way to induce the explant for meristem differentiation and shoot formation is with the use of plant hormones. Hormones can effect a vast array of genetic pathways, with the right hormone concentration balance at the right time it is possible to dirrect the tissue to any differentiation path we desire. The drawback of using hormones is the precise application needed and the broad spectrum of effects it have on the tissue such as epigenetic changes. In order to mitigate those drawbacks, researchers tried to induce regeneration with a less comprehensive effects on the plant by switching on specific genetic pathways with the use of trascription factors. those transcription factors are referred to as developmental regulators (DRs). It has been shown that by over expressing some DRs in the cell it would be possible to differentiate the cells in a more precise manner, similar to the work on induced pluripotent stem cells in humans. Some other work have been done to mitigate other drawbacks of tissue culture like precise growth medium and asceptic enviroment by utelizing the plant own enviroment to create a controled micro enviroment in-planta. Lately Zhang et al. shown promising work on pomelo, in which they intruduce a hybrid method of in-planta and tissue culture, where the transformation took place in a close environment around incision on soil grown plant, with the use of antibiotics as selection and hormones for development induction[@Zhang2017]. This method is revolutionaty in that tissue that is still attached to the plant is much less susceptible to contaminations and most of its micronutrients can come from the neighboring tissues.

### De-novo meristem induction and transformation

Maher et al.[@Maher2020] introduced the next step in independence from tissue culture by combining the two approaches of in-planta transformation and de-novo meristem induction. In that mathod developmental regulators (DRs), CRISPR/Cas9 and sgRNA are delivered to somatic cells of whole plants through Agroinfiltration. This induces meristems that produce shoots solely in cells that were infected successfully with T-DNA bearing CRISPR/Cas9 and sgRNA.

Examples of developmental regulators (DRs):

-   Wuschel2 (_WUS2_) from _Zea mays_ (Maize) is a transcription factor that plays a central role during early embryogenesis, organogenesis and flowering, probably by regulating expression of specific genes. Required to specify stem cell identity in meristems, such as shoot apical meristem (SAM)[@Maher2020].

-   Shoot meristemless (_STM_) from _Arabidopsis thaliana_ appears to function in keeping central meristem cells undifferentiated, thus
playing a major role in maintaining shoot and floral meristems[@Maher2020].

-   Isopentenyl transferase (_IPT_) from the Ti-plasmid of _Agrobacterium tumefaciens_ is a key enzyme in cytokinin biosynthesis[@Maher2020].

-   Wuschel-related homeobox 11 (_WOX11_) from _Populus trichocarpa_ (Poplar tree) acts as master regulator conducting the expression of key  transcription factors to induce de novo shoot organogenesis in poplar[@Liu2018a].

DRs have a long history in plant development research and in recent years are utilized in transformation methods. WUSCHEL is the most known DR that is involved in the meristem cell identity. Several more DRs were identified in their capability to induce cell potency like the tumor induced gene IPT from _Agrobacterium tumefaciens_. Novel methods for the identification of new tissue specific transcription factors and their goal as a DRs in the cell became possible and commonly used, such as tissue specific transcriptome.

### Virus-Induced Gene Editing (VIGE)

VIGE is another example of in-planta transformation method with very different strategy in comparison to the former. It strives to achive systemic gene editing in order to spare the hurdle of regeneration. It has been demonstrated that the CRISPR/Cas system is able to achieve efficient gene editing in plants through either transient expression or transgenic plants. In order to bypasses the requirement for transformation and/or regeneration, Ali et al. developed a virus-mediated genome editing system by separating the two component Cas9 endonuclease and sgRNA. First, a Cas9 overexpressing (Cas9-OE) transgenic plant was generated through Agrobacterium-mediated transformation. Then, modified vector propagates through the plant similar to the VIGS process and edited the target gene[@Ali2015a]. Typically, VIGE approach results in low frequencies of gene editing in somatic cells, and recovery of mutant progeny is rare. Plant phloem exudates contain many transfer RNA-like sequences capable of cell-to-cell movement, which called mobile RNA sequences[@zhang2016trna]. To overcome VIGE setbacks, Voytas et al.[@Ellison2020] fused mobile RNA sequences to the 3-end of the sgRNA to gain better access to the germline. The mobile RNA sequences in question is Flowering LocusT (FT) and tRNAs. FT is transcribed in leaf vascular tissue and then moves to the shoot apical meristem to induce flowering. Their results suggests that RNA mobility sequences increase overall virus accumulation which, in turn, results in higher frequencies of heritable gene editing. 

## Green fluorescent protein (GFP) as a reporter gene for transformation

The green fluorescent protein is a protein that exhibits bright green fluorescence when exposed to light in the blue to ultraviolet range. In this study, it is used as a reporter gene for the agrobacterium transformation.
    
## Poplar as a model organism for trees

*Populus* is a model system for some aspects of tree research for several reasons. It has a rapid growth rate compared to other trees, it is easy to propagate and transform and it has a relatively small genome [@Taylor2002], 45 time smaller than that of pine tree. The target gene that we choose for poplar is the Phytoene desaturase (PDS) gene, because it is a well known reporter gene for gene editing. PDS gene encodes one of the important enzymes in the carotenoid biosynthesis pathway. Knockout of PDS gene disrupt the carotenoid synthesis and results in susceptibility to photobleaching of the chloroplasts, which in turn results in albino and dwarf phenotype. Thus, it is a suitable target for evaluation of gene knockout occurrences.

## Apple tree as a commertial crop

Apple trees are one of the most widely grown fruit tree in the world. In this study, apple plants are used as the commercial application for the transformation method. As a member of the Roseaceae plant family it carries the Self-incompatibility (SI) ferlilization system. SI is a complex system, one out of several mechanisms that prevent plants from self-fertilizing mainly by rejection of the male gametophyte to maintain and increase the genetic variability. Plants have evolved two distinct SI systems, the sporophytic (SSI) and the gametophytic (GSI) systems. In *Malus domestica* (Apple), the GSI system requires the production of female determinants, known as S-RNases, which is produced in the style and penetrate the pollen tube to interact with the male determinants. The penetration of S-RNase into the pollen tube triggering a series of responses involving membrane proteins that inhibits the pollen-tube’s growth process [@DelDuca2019]. Inactivation of S-RNase results in self-compatibility plants [@goldway2012self] hence preventing the need of cross pollination and allowing orchards of a single cultivar (in contrast to SI cultivars that requires cross pollination and at least 
two cultivars that pollinate each other). However, up to date non of apple cultivar are self compatible and following that we choose S-RNase as the target for gene editing.

## Hypotheses

By infecting young plants with _Agrobacterium tumefaciens_ harboring T-DNA that contains DRs, CRISPR/Cas9 and sgRNA, it is possible to achieve de-novo shoot regeneration with knockout at the target gene.

By infecting Cas9-OE plants with viral vector bearing sgRNA targeting PDS endogenes, it would be possible to isolate PDS knockout plant.

By infecting Cas9-OE plants with viral vector bearing sgRNA targeting PDS endogenes fused with mobile RNA sequence, it would be possible to isolate from meristem PDS knockout plant.

## Milstones

### Meristem induction and transformation 

* Contraction of vectors with mix of DRs for shoot regeneration examination.

* Achieve de-novo shoot regeneration as a result of the agro-infiltration.

* Observe mutation in the target sequence. 

### VIGE

* Generate a Cas9-OE poplar plant using Agrobacterium-mediated transformation.

* Contraction of viral vectors with sgRNA.

* Contraction of viral vectors with sgRNA fused with mobile RNA sequence.

* Observe virus propagation.

* Observe mutation in the target sequence.

## Goal

* Develop new method for in-planta trees transformation.

#  Materials and methods

The main goal of this work is to develope a protocol as the final product. For that purpose we use an array of already established protocols for the preperation and validation of our experiments. Some of the protocols use in this work were used as is from the source, but many others were changed to fit our specific use case.

## Plant matirial 

For the experiments in this work, we used two types of plant material. The first is _Populus Alba-Tremula_ (PopAT) cultivar, which were reproduced by vegetative reproduction in tissue culture (Table \ref{tab:Poplar mediums}). The second is _Malus domestica_ (apple) cultivar, from seeds that were extracted from apple fruits.

```{r Poplar mediums, echo=FALSE}

Components = c("","","MS","Sucrose","Agar","NAA","BAP","IBA","Kanamycin","Cefotaxime","Timentin","pH")
M1 = c("(Plant Propagation","and Pre-Culture)","4.43 g/L","30 g/L","5.8 g/L","","","","","","","5.8")
M2 = c("(Co-Culture)","","4.43 g/L","30 g/L","5.8 g/L","0.05 mg/L","0.5 mg/L","","","","","5.8")
M3 = c("(Shoot Induction)","","4.43 g/L","30 g/L","5.8 g/L","0.05 mg/L","0.5 mg/L","","30 mg/L","200 mg/L","200 mg/L","5.8")
M4 = c("(Root Induction)","","2.215 g/L","30 g/L","5.8 g/L","0.02 mg/L","","0.05 mg/L","30 mg/L","200 mg/L","200 mg/L","5.8")


mediums = data.frame(Components = Components, M1 = M1, M2 = M2, M3 = M3, M4 = M4)

sep = c(rep('',length(Components)))                             
kable(mediums, caption = "Composition of media for cultivation, tranformation, selection and regenetarion of hybrid poplar P.alba X P.glandulosa",booktabs = TRUE, align = "ccccc", linesep = sep) %>% kable_styling(latex_options = "HOLD_position")
```
## Matirials

* [Petri-dish](https://shop.gbo.com/en/row/products/bioscience/microbiology-bacteriology/dishes/petri-dishes/) (Greiner Bio-One™)
* [Magenta™ vessel](https://www.sigmaaldrich.com/IL/en/substance/magentavessel1234598765)
* PARAFILM® (Sigma-Aldrich™)
* [Lysogeny broth](https://www.formedium.com/product/lb-broth-lennox/) (LB from ForMedium™)
* [Agar](https://www.formedium.com/product/agar/)
* Murashige & Skoog (MS from Sigma-Aldrich™)
* [CTAB](https://www.sigmaaldrich.com/catalog/product/sigma/h6269?lang=en&region=IL) (Sigma-Aldrich™)
* [TRI reagent®](https://www.sigmaaldrich.com/IL/en/technical-documents/protocol/protein-biology/protein-lysis-and-extraction/tri-reagent?gclid=CjwKCAiA_omPBhBBEiwAcg7smZ6kLByCZQUwUDAqDPPMJG-_DgYEHC8fwC_oR83txhumwBdGseVw9BoCBpsQAvD_BwE) (Sigma-Aldrich™)
* Acetosyringone (Sigma-Aldrich™)

### antibiotics

* Ampicillin (Sigma-Aldrich™)
* Kanamycin (Sigma-Aldrich™)
* Rifampicin (Sigma-Aldrich™)
* Gentamicin (Sigma-Aldrich™)
* Spectinomycin (Sigma-Aldrich™)

### Kits

[Miniplsmid putification kit](https://www.geneaid.com/product.php?act=view&id=180) (Presto™)

### Instruments

* [MicroPulser Electroporator by Bio-Rad](https://www.bio-rad.com/en-il/product/micropulser-electroporator?ID=83527990-34fb-4b33-b955-ca53b57bf8b9)
* [Microcentrifuge](https://www.thermofisher.com/order/catalog/product/69720#/69720) (Pierce™)
* PCR machine [Biometra TRIO](https://www.analytik-jena.com/products/life-science/pcr-qpcr-thermal-cycler/thermal-cycler-pcr/biometra-trio-series/)
* Power supply + Electrophoresis bath ([PowerPac™ Basic Power Supply](https://www.bio-rad.com/en-il/product/powerpac-basic-power-supply?ID=bea5dea1-cef0-43ad-8af5-b2c0287f6e07), [Wide Mini ReadySub-Cell GT Horizontal Electrophoresis System](https://www.bio-rad.com/en-il/sku/1704489-wide-mini-readysub-cell-gt-horizontal-electrophoresis-system-for-precast-gels?ID=1704489))
* UV-light table [($UV_{DOC}$ HD2 by Uvitec Cambridge)](https://topac.com/documents/geldoc_%20catalogue.pdf)

## Protocols

* [Gibson Assembly Cloning](https://www.addgene.org/protocols/gibson-assembly/) [@Gibson2009]: In 2009 Dr. Daniel Gibson and colleagues at the J. Craig Venter Institute developed a novel method for the easy assembly of multiple linear DNA fragments. Regardless of fragment length or end compatibility, multiple overlapping DNA fragments can be joined in a single isothermal reaction. With the activities of three different enzymes, the product of a Gibson Assembly is a fully ligated double-stranded DNA molecule. This has proven to be an efficient and effective method for the assembly of plasmids, and molecular biologists now use this method extensively.

* [Golden-Gate cloning](https://blog.addgene.org/plasmids-101-golden-gate-cloning)[@Cermak2017]: Golden Gate cloning technology relies on Type IIS restriction enzymes, first discovered in 1996. Type IIS restriction enzymes are unique from "traditional" restriction enzymes in that they cleave outside of their recognition sequence, creating four base flanking overhangs. Since these overhangs are not part of the recognition sequence, they can be customized to direct assembly of DNA fragments. When designed correctly, the recognition sites do not appear in the final construct, allowing for precise, scarless cloning.

* Gel agarose for DNA electrophoresis[@Lee2012]: Agarose gel electrophoresis is the most effective way of separating DNA fragments of varying sizes ranging from 100 bp to 25 kb. During gelation, agarose polymers associate non-covalently and form a network of bundles whose pore sizes determine a gel's molecular sieving properties. To separate DNA using agarose gel electrophoresis, the DNA is loaded into pre-cast wells in the gel and a current applied. The phosphate backbone of the DNA (and RNA) molecule is negatively charged, therefore when placed in an electric field, DNA fragments will migrate to the positively charged anode. Because DNA has a uniform mass/charge ratio, DNA molecules are separated by size within an agarose gel in a pattern such that the distance traveled is inversely proportional to the log of its molecular weight.

* E.coli Heat-shock competent preperation and [transformation](https://www.addgene.org/protocols/bacterial-transformation/)[@Chang2017]: Calcium chloride heat-shock transformation is a powerful molecular biology technique used to introduce foreign DNA into a host cell. The concept of the technique is to encourages bacterial cells to uptake DNA from the surrounding environment when stressed by heat shock by using CaCl2 to render them more competent. This technique allow for introduction of plasmid for bacterial transformation.

* CTAB protocol for the isolating of DNA from plant tissues[@Porebski1997]: A relatively quick, inexpensive and consistent protocol for extraction of DNA from plant tissues. CTAB extraction, employing high salt concentrations to remove polysaccharides, the use of polyvinyl pyrrolidone (PVP) to remove polyphenols, an extended RNase treatment and a phenol-chloroform extraction.

* TRI reagent® protocol for the isolation of RNA, DNA or proteins[@Chomczynski1993]: method for simultaneous isolation of RNA, DNA and proteins from cell and tissue samples.


# Experimants scheme

Experimants scheme as it was planned for the achievment of the milestones and to test the hypotheses.

```{r Experimants scheme, echo=FALSE}
library(DiagrammeR)
library(DiagrammeRsvg)  # for conversion to svg
library(rsvg)  # for saving svg

FlowChart <- 
  grViz(diagram = "digraph flowchart {
    node [fontname = arial, shape = rect];
    edge [fontname = arial];

    hypothesis [label = 'Hypothesis: \nBy infecting young plants with Agrobacterium tumefaciens \nharboring T-DNA that contains DRs, CRISPR/Cas9 and sgRNA, \nit is possible to achieve de-novo shoot regeneration with knockout at the target gene.']

    plasmids [label = 'Construction of plasmids containing DRs, Cas9 and sgRNAs \n11 per cultivar']

    infection_methods [label = 'Infection methods experiment:\n3 Infection methods with all 11 plasmids']

    DRs [label = 'DRs experiment:\nBest Infection methods with all 11 plasmids']

    ScaleUp [label = 'Scale-up experiment:\nBest Infection methods with most successful plasmids']
    
    hypothesis -> plasmids -> infection_methods -> DRs -> ScaleUp;
  }")
  
tmp <- capture.output(rsvg_png(charToRaw(export_svg(FlowChart)),'De-novo_meristem_induction_FlowChart.png'))

FlowChart <- 
  grViz(diagram = "digraph flowchart {
    node [fontname = arial, shape = rect];
    edge [fontname = arial];

    hypothesis [label = 'Hypothesis: \nBy infecting Cas9-OE plants with viral vector bearing sgRNA targeting PDS endogenes, \nit would be possible to isolate PDS knockout plant.']

    plants [label = 'Generating Cas9-OE tree']

    vectors [label = 'Generating a viral vector bearing sgRNA targeting PDS endogenes']

    NB_infection [label = 'Infecting Cas9-OE Nicotiana Benthamiana with GM-vector']

    plant_infection [label = 'Infecting Cas9-OE tree with GM-vector']
    
    hypothesis -> plants -> vectors -> NB_infection -> plant_infection;
  }")
  
tmp <- capture.output(rsvg_png(charToRaw(export_svg(FlowChart)),'VIGE_FlowChart.png'))
```
```{r VIGE_FlowChart, out.width = "80%", fig.align="center", echo=FALSE, fig.cap="VIGE experimants scheme"}
include_graphics("VIGE_FlowChart.png") 
```
```{r De-novo_meristem_induction_FlowChart, out.width = "80%", fig.align="center", echo=FALSE, fig.cap="De-novo meristem induction experimants scheme"}
include_graphics("De-novo_meristem_induction_FlowChart.png") 
```

## Implementing VIGE in trees

### Generating Cas9-OE poplar plant

Based on protocol developed by Song et al.[@Song2019] with minor changes.

Top10 bacteria with $3\Omega1$-NPT-ubi::Cas9 plasmid are grown in LB liquid-medium containing 50 mg/L Spectinomycin, at 200rpm over-night at $37^oC$. The following day the plasmids are extracted using miniplsmid putification kit. Then the plasmid are introduced into EHA-105 agrobacterium using electropuration and plated on LB-agar medium containing 50 mg/L Spectinomycin and 50 mg/L Gentamicin, at $28^oC$ for two days. A single colony is picked to LB liquid-medium containing 50 mg/L Spectinomycin and 50 mg/L Gentamicin and incubated with shaking 200rpm over-night at $28^oC$. The cells are then harvested by centrifuge at 5000 rpm for 10 min and resuspended with 1/2 MS solution (pH 5.8 to 6.0) containing 5% (w/v) sucrose and acetosyringone (100 µM) as the transformation solution, to a culture density of 0.6 $OD_{600}$. 

The leaf explants for transformation are prepared from 3-week-old plantlets that were cut perpendicular to main veins and layed on M1 for 2 days. Next the cut leaves are incubated with the suspended agrobacterium for 15 min with slow shaking every 5 min and then blotted with sterile filter paper to remove the excess bacteria and cultured on M2 in dark conditions for 3 days.

The co-cultivated leaves are washed X2 in sterilized distilled water for 5 min, blotted with sterile filter paper to remove the excess water and transferred to M3 for 10 days. After 4 weeks 1 to 2 cm shoots are cut off and transferred to M4 For obtaining plants. The transgenic poplar plants were transferred to M1 for 3 weeks to avoid agrobacterium contamination.

### Generating a viral vector bearing sgRNA targeting PDS endogenes fused with mobile RNA sequence and GFP.

Following Lee et al.[@Lee2015] with minor changes.

- Cunstraction of plasmids containing virus genome segments.

BSMV:\
- pCaBSMV-$\alpha$\
- pCaBSMV-$\beta$\
- pCaBSMV-$\gamma$::PDS.gRNA\

TRV:\
- pRNA1\
- pRNA2.PEBV::PDS.gRNA\

The TRV2 and $\gamma$ partite cloned with sgRNA via ligation independent cloning (LIC). GV-3101 agrobacterium is transformed using electropuration and grown on petri-plate containing  LB-agar medium containing 50 mg/L kanamycin and 10 mg/L Gentamicin, at $28^oC$ for two days. A single colony is picked to LB liquid-medium containing 50 mg/L kanamycin and 10 mg/L  Gentamicin, and incubated with shaking 200rpm over-night at $28^oC$. Cells are collected by centrifuge for 10min in 4500rpm, and re-suspended in infiltration buffer(1/4 MS with 1% sucrose, 100 $\mu M$ and 0.01% Silwet L-77) to $OD_{600}$ of 0.5.

PDS-gRNA and GFP are cloned into $pPEBV-TRV_2$. The mobile RNA sequence is Isoleucine-tRNA.

### Infecting Cas9-OE *Nicotiana Benthamiana* with GM-vector

Cas9-OE *Nicotiana Benthamiana* is infected with GM-vector as a Preliminary experiment, for examination of the GM-vector. Bacteria is infiltrated into *Nicotiana Benthamiana* leaves by 1 mL plastic syringe (without needle), dry in the light (1 hour) and keep in the dark for 24 hours, in $23-25^oC$. The plants are than moved to a long day conditions (16-8 light/dark), in $23-25^oC$ for 3-4 days. The uninfected leaves are collected and RNA is isolated using TRI reagent protocol. The RNA was than reverse transcribe to cDNA. A section of the vector was
Amplified in PCR (Gradient thermal cycler by VWR) for propagation examination. Wild-Type *Nicotiana Benthamiana* was also infected with GM-vector as a propagation factory, to generate GM-vector from T-DNA, in the same manner.

### Infecting Cas9-OE poplar and Cas9-OE apples with GM-vector

Infiltrated leaves from positive plants are collected and immediately transferred to liquid nitrogen. The tissue is grounded to a fine powder and suspended in phosphate buffer (0.1 M, pH 7.5) solution (3 mL per gram of tissue-powder). Sea-sand is added to induce micro-cuts to the leaves. The given liquid is rubbed on the leaves of young Cas9-OE plants to induce viral infection.

## Implementing De-novo meristem induction and transformation.

### Construction of plasmids containing DRs, Cas9 and sgRNAs

Constructing plasmids with combinations of DRs for de-novo maristem induction for novel transformation protocol.

All the plasmid building blocks for the final vectors are bought from [Addgene](https://www.addgene.org/) (Table \ref{tab:Building blocks}), except for pMOD-B with S-RNase sgRNA array and pMOD-C with WOX11 from _Populus trichocarpa_ which was assembled in-house.

```{r Building blocks, echo=FALSE}

Name = c("pTRANS_221","","pMOD_B2103","pMOD_C'5014","pMM107","pRN110")
Purpose = c("Empty Backbone with Cas9-csy4 gene","and kana resistance","cassette for cloning multiple gRNA","Module C' with Pnos::WUS2","Module C' with 35S::IPT","Module D' with CmYLCV::STM")
BackBone = c("pCAMBIA","","pMOD_B2000","pMOD_C'4800EC","pMOD_C'5014","pMOD_D4800EC")
Insert = c("None","","None","WUS2","IPT","STM")
Species = c(" ",""," ","Maize","A.tumefaciens","A.thaliana")
Number = c(91115,"",91061,127219,127227,127228)


building_blocks = data.frame(Name = Name, Purpose = Purpose, BackBone = BackBone, Insert = Insert, Species = Species, Number = Number)
                             
sep = c(rep('',length(Name)))
kable(building_blocks, caption = "Building blocks from Addgene",booktabs = TRUE, linesep = sep) %>%
  kable_styling(latex_options = "HOLD_position")
```

* **Species** column refers to the insert's origin species.

For the assembly and validation off the final vectors, costume primers were used as detailed in Table \ref{tab:Primers}.

```{r Primers, echo=FALSE}
primers <- read.csv("Tables/primers.csv")

sep <- c()
for (row in 1:(nrow(primers)-1)) {
  if (primers[row+1,"Purpose"] == ""){
    sep <- append(sep,'')
  }
  else{
    sep <- append(sep,'\\addlinespace')
  }
}

kable(primers, caption = "All primers" ,booktabs = TRUE, linesep = sep) %>%
  kable_styling(latex_options = "HOLD_position")
```

The gene _WOX11_ from _Populus trichocarpa_ were synthesized into pUC57 plasmid as a service from [GenScript](https://www.genscript.com/)(New Jersey, USA) and were assembled into pMOD-C backbone using [Gibson Assembly Cloning](https://www.addgene.org/protocols/gibson-assembly/) protocol[@Gibson2009] (Table \ref{tab:Primers}).

The final plasmids were constructed into pTRANS backbone using [Golden-Gate cloning](https://blog.addgene.org/plasmids-101-golden-gate-cloning)[@Cermak2017] (Table \ref{tab:AllConstracts}).

Transformation of Top10 bacteria with constructs for replication was preformed using [heat-shock transformation](https://www.addgene.org/protocols/bacterial-transformation/) and growing on petri-dish containing Lysogeny broth medium with 1.1 % Agar 50 mg/L Ampicillin, at $37^oC$ over-night. Evaluating constructs using colony PCR (Polymerase chain reaction) reaction (Table \ref{tab:Primers}). Growing positive colonies in LB liquid-medium (containing 50 mg/L Ampicillin), shaking 200 rpm over-night at $37^oC$. Purifying constructs using miniplsmid putification kit. 

Transformation of EHA-105 agrobacterium using electroporation MicroPulser Electroporator and growing on petri-dish containing LB-agar medium with 50 mg/L Ampicillin and 50 mg/L Rifampicin at $28^oC$ for 2 days. Positive colonies are validated again using colony PCR. 

Cultures of each positive EHA-105 agrobacterium are spread on new petri-dish containing LB-agar medium with 50 mg/L Ampicillin and 50 mg/L Rifampicin at $28^oC$ for 4 days. At the day of plant transformation, the whole plate is picked using Drigalski spatula and transferred to 2 ml Microcentrifuge with a tip. The transferred colonies are weighed and elut in 2 $\mu l$ per mg Liquid medium containing $1/2$ MS, 1 % sucrose and 200 $\mu M$ acetosyringone.

### Infection methods experiment

The first experiment is designed for the assessment of different infection methods on the agro-infection effectiveness.
Seeds are extracted from apple of the variety Pink-lady, that have been refrigerated for few months. The seeds are then grown in a germination tray (11X17 cells) for 1 month.

Three methods of Agro-infection was tested on the plants. The first is agro-infiltration to the leaves through pressure with needleless syringe. The second is micro-stabs of concentrated bacteria culture into the leaf vains. The last is vertical cut of the stem as far as possible from the nearest axillary bud below and injection of concentrated bacteria culture into the cut (Figure \ref{fig:Agro-infection methods}). Each infection method got repetitions with each construct as follows: first 4, second 3, third 1. 


```{r Agro-infection methods, out.width = "100%", fig.align="center", echo=FALSE, fig.cap="Infection of Apples with A.tumefaciens, three methods. \\newline{} \\textbf{A.} agro-infiltration to the leaves \\textbf{B.} micro-stabs to the leaf vains \\textbf{C.} injection to the stem cut"}
include_graphics("apple_injection3.png") 
```

### DRs experiment

The second experiment is designed for the assessment of different DRs on the de-novo shoot induction influence.
Seeds are extracted from 4 varieties of apples that have been refrigerated for few months (Pink Lady, Granny Smith, Starking, Golden Delicious). The seeds are then grown in a germination tray (11X17 cells) for 1 month.

All plant are infected as described in method C (Figure \ref{fig:Agro-injection}). After injection, all stem cuts are covered with PARAFILM for 1 week. New shoots that grow from axillary buds are removed once a week.

```{r Agro-injection, out.width = "100%", fig.align="center", echo=FALSE, fig.cap="Injection of A.tumefaciens into a stem cut of 4 apple Varieties. \\newline{} \\textbf{A.} Injection into a stem cut \\textbf{B.} Starking \\textbf{C.} Pink Lady \\textbf{D.} Golden Delicious \\textbf{E.} Granny Smith"}
include_graphics("apple_injection2.png") 
```


Every construct have 11 repetitions in each apple variety, 44 overall.

Validation of cassette integration in the genome by DNA extraction from old leaf in the base of the plant and new leaf in the shoot using CTAB protocol and sequence amplification from the T-DNA cassette that is part of the Cas9 gene using specific primers in PCR reaction. The PCR products are analyzed using agarose gel electrophoresis, and photographed on a UV-light table (Table \ref{tab:Primers}).

Validation of mutation in the targeted sequence is achieved by amplification using specific primers in PCR reaction and sanger sequencing as a service from [(Macrogen Europe B.V., Meibergdreef 57 1105 BA, Amsterdam, the Netherlands)](https://dna.macrogen-europe.com/eng/index.jsp) (Table \ref{tab:Primers}).

### Scale-up experiment
The third experiment is designed for the assessment of de-novo shoot induction occurrences influenced by the two most promising DR combinations. Seeds are extracted from apples of the variety Pink-lady and Granny-smith, that have been refrigerated for few months. The seeds are then grown in a small pots (5X5 cm wide and 7 cm high) for 1 month. 800 seeds extracted of 2 varieties are infected with 2 most promising plasmids and 1 without DRs. Third of the plants are infected with the plasmid *WOX11*-*STM*, third with *WUS2*-*STM* and third with Empty-Empty. The plants are then sealed with parafilm and aluminum foil. The aluminum foil are covering for 2 days, and the parafilm for additional week.

### High humidity experiment
Another experiment is designed to isolate as much of the variability in soil experiment by growing the seedlings in a semi-sterile and close environment. Than the plants can be infected in the same manner only without covering the infected stem cut.  

### Poplar infections
Similar experiments were performed on Poplar plants, the main difference was that the plant material was started from vegetative reproduction and not from seeds.  

Another experiment was the infection of poplar ex-plant on sterile growth medium without the use of hormones and antibiotics for selection.

### Identification of new trascription factors that are development regulators in poplar.
Tissue specific transcriptomes of the hybrid *Populus Alba-Tremula* (PopAT) were obtaind from [NCBI's SRA site](https://www.ncbi.nlm.nih.gov/bioproject/?term=(Populus%20tremula%20x%20alba,%20clone%20717%20Transcriptome)%20AND%20bioproject_sra[filter]%20NOT%20bioproject_gap[filter]). The tissues that were analyzed were shoot-tip, root-tip, bud, callus, xylem, leaf and bark (RNA-seq tissues). The transcripts were analyzed in order to identify new trascription factors that are most representative of specific tissue, and therefor suspected to have a strong role as a development regulators for that specific tissue in poplar. The transcripts were downloaded as an SRR files and was converted to FASTQ using the sratoolkit [(version 2.11.0)](https://trace.ncbi.nlm.nih.gov/Traces/sra/sra.cgi?view=software). Than the reads were processed using the [FASTP tool](https://github.com/OpenGene/fastp). After processing, the reads from each tissue were aligned to a reference trascriptome from _Populus trichocarpa_ (PopTri) from [NCBI's Datasets site](https://www.ncbi.nlm.nih.gov/datasets/genomes/?txid=3689&term=Populus&utm_source=gquery&utm_medium=referral&utm_campaign=:assemb) using Burrows-Wheeler Aligner (BWA, version 0.7.1), and was outputted as a SAM file. The alignment files were analyzed further in order to extract the frequency of each transcript occurrences per tissue. The mathematical way we chose to calculate the a score for the representativeness of transcript to a specific tissue was to divide his frequency in the tissue in question against the frequency in each tissue and then look only on the genes that were at least 1 order of magnitude grater in the tissue in question in comparison to all other tissues. Then we sort by the sum of all ratios.

All the analysis code is available on GitHub in [https://github.com/BenSiv/PopAT-expression-analysis](https://github.com/BenSiv/PopAT-expression-analysis).

### Validation of the RNA-seq analysis with real-time quantitive PCR (RT-qPCR)
For the validation of the RNA-seq analysis, RNA is exctarcted from the same tissues as the RNA-seq tissues excluding callus (RT-qPCR tissues). Then RT-qPCR is preformed on the complementary DNA (cDNA) with specific primers for the gene that was identified as the most shoot-tip specific by the RNA-seq analysis. Several normalizer genes were used for the quantification of the expression between the different tissues.


# Results

In this work I attempted to implement a novel gene editing method in-planta on trees. I worked on two species, Poplar from vegetative reproduction in tissue culture and apples plantlets from seeds. Poplar was used since it is a model plant and could serve for the best case scenario for gene editing in trees. I used PopAT so that I could control and mitigate as much of the variability in the experiments, because there is no genetic diversity in the plants and as a model organism, they are known to be susceptible to agroinfiltration. With apples it was different, the plants were grown from seeds and since apples are self-incompatible, the seedlings are diverse in their genetic makeup. Although the research is based on some success of others, still there is no consensus about the best practice to approach in-planta transformation. In the case of VIGE, I tried to generate Cas9-OE poplar plants for infection and teamed up with another research group for the assement of the system on Cas9-OE apple plants. In the case of de-novo maristem induction and transformation, I examined new plants and new DRs, therefore however I did not begin by expressing the genes in tissue culture since I didn’t reasoned that as my final goal was in-planta transformation and because, it is well documented that one cannot deduce from tissue culture results to an in-planta experiment in a straightforward manner. Hence, the experiment scheme starts with in-planta experiments. I began by assessing the best approach to facilitate the infiltration of the agrobacterium by different mechanical interventions. Next, I examined the influence of different combinations of DRs on the plant development. Finally, I scaled up the best results from the last two experiments for statistical analysis. In parallel I tried to examine the natural expression of genes in different tissues and especially in the shoot-tip by analyzing tissue specific transcriptomes of PopAT.


## Regeneration of sterile PopAT

For all the experiments envolving poplar plants, we had to obtain enough plant metirial. For that purpose we reproduced poplar plants in tissue culture through regeneration proccess (Figure \ref{fig:PopAT-regeneration}).

```{r PopAT-regeneration, out.width = "100%", fig.align="center", fig.pos="H",, echo=FALSE, fig.cap="Sterile Populus alba x tremula A: Whole plant, B: Leaves with vertical incision, C: Shoots"}
include_graphics("PopAT-regeneration.png") 
```
## Implementing VIGE in trees

### Generating of Cas9-OE poplar plant

PopAT were transformed as described in [**Generating Cas9-OE poplar plant**](#Generating Cas9-OE poplar plant) in order to generate Cas9-OE poplar plants for purpose of implementing VIGE on them. Many attempts were made on that goal, most of which endded in failure as results of fungi contamination or agrobacterium over growth. One attempt even got to the point of final transformant validation (Figure \ref{fig:PopAT-transformants}). 

```{r PopAT-transformants, out.width = "50%", fig.align="center", fig.pos="H", echo=FALSE, fig.cap="Regenerated shoots T2 and T3 are susceptible to kanamycin, T4 is partially susceptible and T1 is unaffected"}
include_graphics("T1-4.jpg") 
```

DNA was isolated from these shoots and PCR was preformed with primers from the T-DNA in order to validate transformation. No shoot was found positive.

### Generating a viral vector bearing sgRNA targeting PDS endogenes fused with mobile RNA sequence and GFP.

```{r Viruses, echo=FALSE}
VVectors <- read.csv("Tables/Viral_vectors.csv")

kable(VVectors, caption = "Viral vectors" ,booktabs = TRUE) %>%
  kable_styling(latex_options = "HOLD_position")
```

```{r Viral-vectors, out.width = "100%", fig.align="center", fig.pos="H", echo=FALSE, fig.cap="Viral constructs designs. \\textbf{A.} Fast cloning cassette for guidRNAs. sgPRO - sub-genomic promoter. crRNA - conserved part of CRISPR guidRNA. \\textbf{B.} TRV two sub-genomes constructs. \\textbf{C.} BSMV three sub-genomes constructs. (Figures were adjusted from Yuan et al., 2011)"}
include_graphics("VIGE_vectors.png") 
```


### Infecting Cas9-OE poplar and Cas9-OE apples with GM-vector

Since we couldn't successfully generate Cas9-OE poplars, we couldn't infect them with our vectors. Fortunately, we teamed up as part of a graiter collaboration with Moshe Flaishman's lab from the Department of Fruit Trees Sciences in the Institute of Plant Sciences at The Volcani Center. Flaishman's lab generated two lines of Cas9-OE apples (#166, #177). These plants where transformed with the plasmid **pK7WGF2,0::hCas9** using Agrobacterium mediated transformation in tissue culture (Figure \ref{fig:Flaishmans_apples}).  


```{r Flaishmans_apples, out.width = "100%", fig.align="center", echo=FALSE, fig.pos= "H", fig.cap="One of the plants that reproduced from the \\#167 line. \\newline{} \\textbf{A.} Map of the T-DNA vector \\textbf{B.} The plant post infection"}
include_graphics("Flaishmans_apples.png")
```

As part of the collaboration, some other efforts were made to remove the T-DNA cassette from the genome by inducing multiple digestion sites at the borders of the cassette. In order to examine the possibility of T-DNA editing by VIGE in apple trees, we constructed two types of viruses – TRV and BSMV – with guides against different targets in the borders of the T-DNA cassette, which harbor the Cas9 gene and already exist in transgenic apple plants (Figure \ref{fig:Cassette_map}).

```{r Cassette_map, out.width = "100%", fig.align="center", echo=FALSE, fig.pos= "H", fig.cap="Map of binary plasmid used in transgenic apple lines \\#166 and \\#167. Green squares - T-DNA borders, arrow heads - targets for viral guide-RNAs, RB - right border, LB - left border."}
include_graphics("Cassette_map.png")
```

Viruses were propagated in N.benthamiana plants by Agroinfiltration of the plasmids. Systemic infection of the viruses was confirmed in un-infected N.benthamiana leaves. In this first attempt, we were able to verify the infectious of two BSMV and four TRV viruses against different targets (Table \ref{tab:apple_infected}). We infected each virus on two plantlets of each of the apple transgenic lines (#166, #167). Out of the 24 infected plantlets only 18 survived. 

```{r apple_infected, echo=FALSE}
infected_apples <- data.frame(Treatment_number = c(1,2,3,4,5,6,7),
                              Virus = c("TRV", "TRV", "BSMV", "TRV", "TRV", "BSMV", "Negative control"),
                              Target = c("LB1","LB2","LB1","RB2","RB1","RB1","-"),
                              Plantlets_infected = c(4,4,4,4,4,4,4),
                              Plantlets_survived = c(3,4,4,3,3,1,4))

kable(infected_apples, caption = "Viral infection of apple plantlets" ,booktabs = TRUE, linesep = "") %>%
  kable_styling(latex_options = "HOLD_position")
```
One month post infection we examined systemic infection of the virus in the plantlets by PCR on new emerged leaves. Out of the 18 plantlets checked, only five showed systemic infection: three were TRV positive and two BSMV positive (Figure \ref{fig:Apples_infection_validation}). 

```{r Apples_infection_validation, out.width = "100%", fig.align="center", echo=FALSE, fig.pos= "H", fig.cap="PCR on new leaves of infected apple plantlets with virus specific primers. Upper gel - TRV, Lower gel - BSMV. '+' - plasmid positive control. 166/167 – apple transgenic lines."}
include_graphics("Apples_infection_validation.jpg")
```

Since some of the plantlets showed un-expected virus infection (TRV instead of BSMV), probably due to cross infection or contamination, we decided to check all targets in all plantlets
for editing events. We amplified target regions from genomic DNA and sent them for Sanger sequencing. In one of the plants, which was infected with TRV, harboring the LB2-gRNA, we found a double chromatogram starting around the PAM site of the LB2 target (Figure \ref{fig:Double_chromotogram}). 

```{r Double_chromotogram, out.width = "100%", fig.align="center", echo=FALSE, fig.pos= "H", fig.cap="Sequencing of the LB2 region in infected apple plants. Upper chromatogram - Sanger sequencing of LB2 target in plant infected by TRV-LB2gRNA. Lower chromatogram - Sanger sequencing of LB2 target in plant infected by TRV-RB1gRNA. Red arrow – start of alternate sequence. Turquoise bar - LB2 target."}
include_graphics("Double_chromotogram.png")
```

Analyzing of the alternative chromatogram predicts a deletion of 31 bps and an insertion of +A. additional sequencing is required to determine the frequency of this mutation and to search for additional low frequencies InDels.

### Improving gRNA delivery by adding tRNA sequence to the gRNA

In order to improve the systemic infection in apple, we purchased the new viral system developed by Dan Voytas lab who showed efficient editing in model plants and crops [@Ellison2020]. In this system, a tRNA sequence which was added to the gRNA showed improved mobility and editing of meristems. We prepared new constructs with guides against PDS gene of apple to check the feasibility of VIGE in trees with a visible marker. In addition, we constructed TRV-GFP-tRNA plasmids in order to compare the infection abilities of the two systems. We propagated the TRV-GFP-tRNA virus in _N.benthamiana_ and infected young apple seedlings by stabbing one leaf. Two weeks post infection we found a stronger GFP signal in the TRV-GFP-tRNA infected plants compare to TRV-GFP (Figure \ref{fig:TRV_GFP}).

```{r TRV_GFP, out.width = "100%", fig.align="center", echo=FALSE, fig.pos= "H", fig.cap="GFP fluorescence in apple cotyledons two weeks after viral infection."}
include_graphics("TRV_GFP.png")
```

Following the improvement in GFP expression, we designed an experiment to check if VIGE can also be improve. We constructed a TRV-PDSgRNA-tRNA virus, targeting the PDS endogene of apple. As before, we infected apple plantlets by co-cultivation and checked for viral infection in new un-infected leaves. This time we could not find any evidence for virus presence in the new leaves. Moreover, most of the plantlets died few weeks after infection. Hence, this experiment needs to be repeated to obtain results that are more conclusive. 


## Implementing De-novo maristem induction and transformation in trees

### Construction of plasmids containing DRs, Cas9 and sgRNAs
A variety of plasmids was required to insure a wide range of DR combinations on both cultivars with different reporter genes. Therefor as much as 21 plasmids constructed (Table \ref{tab:AllConstracts}). The construction of the plasmid was planned with the golden gate assembly protocol. This assembly method is based on the compatibility of a backbone and 4 modules. Each module can either include or not include a functional insert. The combination of the different building blocks provides the diverse array of plasmids. Some of the building blocks were purchased ready-made and some I constructed.


```{r AllConstracts, echo=FALSE}
AllConstracts <- read.csv("Tables/AllConstracts.csv")

sep <- c()
for (row in 1:(nrow(AllConstracts)-1)) {
  if (AllConstracts[row+1,"Target"] == ""){
    sep <- append(sep,'')
  }
  else{
    sep <- append(sep,'\\addlinespace')
  }
}

kable(AllConstracts, caption = "De-novo maristem induction constracts" ,booktabs = TRUE, linesep = sep) %>%
  kable_styling(latex_options = "HOLD_position")
```

### Infection methods experiment
Assessment of infection methods for the infiltration and infection of apple plants by agrobacterium, was performed by 3 methods (Figure \ref{fig:Agro-infection methods}). Infection method A is based on a well-established protocol for Nicotiana benthamiana where the agrobacterium is driven by pressure into the intercellular spaces in the leaf via the stoma (**Citation**). It should be noted that in contrast to Nicotiana benthamiana, apple leaves are stiff and their ability to absorb liquid is lower and therefor this method may not be as suitable for them. In Infection method B, the agrobacterium is injected into the leaf veins. The reason for to targeting specifically the leaf veins was since in tissue culture those cells are the first to regenerate. In Infection method C, the shoot tip was sniped off and the agrobacterium was injected into the exposed vascular tissue. This method mimics in a way the natural growth of the shoot tip and therefore theoretically increases the possibility of shoot regeneration similarly to the occurrence in grafting. In each infection methods all 11 apple constructs were used (Table \ref{tab:AllConstracts}). Methods A and B were preformed 7 and 6 biological replicates respectively And Method C was performed only once per construct. In infection methods A and B, even after a month no phenotype was observed. However, In contrast in Infection method C, 2 newly formed shoots were observed from the stem cut site two weeks post infection. Those plants were infected with the constructs that included the genes *WOX11*-*STM* and *WUS2*-*STM* (Figure \ref{fig:wox-stm transformant}). In one of the cases, a cotyledon like tissue emerged from the cut beneath the shoot.

```{r wox-stm transformant, out.width = "100%", fig.align="center", echo=FALSE, fig.pos= "H", fig.cap="Injection of A.tumefaciens into a stem cut of apple plantlets from Pink-lady variety. \\newline{} \\textbf{A.} A cotyledon like that emerged from a stem cut, two weeks post infection \\textbf{B.} A shoot grow, three weeks post infection."}
include_graphics("wox-stm transformant3.png") 
```

DNA was extracted from an old leaf in the base of the plant and a new leaf in the shoot. For validation of T-DNA integration in the genome, a segment of the T-DNA cassette that contains the Cas9 gene was amplified by PCR. The PCR products was run on agarose gel and photographed on a UV-light table. For the positive control served the plasmid that was used for the transformation, pTRANS WOX11-STM, and for the negative control served the PCR reaction solution without DNA (Cas9 stock). To validate that the DNA extraction was successful, a sequence from the actin gene was amplified using PCR reaction and was anelyzed too by agarose gel electrophoresis. For negative control the PCR reaction solution without DNA (Actin stock) was used. A band was observed in the new leaf sample at the expected length (201 bp), as seen in the positive control sample and wasn't observed in the old leaf. Both DNA samples showed successful amplification of the Actin gene (Figure \ref{fig:Cas9 gene in transformant}).  

```{r Cas9 gene in transformant, out.width = "80%", fig.align="center", echo=FALSE, fig.pos= "H", fig.cap="Analysis of the presence of Cas9 in DNA extracted from ‘New’ and ‘Old’ leaves of a transformed apple. \\newline{} \\textbf{A.} T-DNA cassette validation with Cas9 primers \\textbf{B.} Control for the presence of DNA in the reactions presented in A, PCR was performed with primers for the actin gene."}
include_graphics("WOX-STM transformant Cas9+Actin3.jpg") 
```
All the apple constructs consist of sgRNA targeting the S-RNse gene (Table \ref{tab:AllConstracts}) and all the apple varieties used carry this gene. Both alleles of the S-RNase gene (S2 and S3) were amplified and sequenced for mutation analysis. No mutation was observed.  

### DRs experiment  
Assessment of DRs for shoot induction in apple plants was performed with all 11 constructs (Table \ref{tab:S-RNase Constracts}), on 4 apple varieties (Starking, Pink Lady, Golden Delicious and Granny Smith) and with 11 repeats per construct (Figure \ref{fig:Agro-injection}). Out of the 352 plants infected, 4 de-novo shoot regeneration were observed, 3 of them were of the Pink-lady variety and the $4^{th}$ was of Starking. The shoot regeneration was observed already after 1 week. (Figure \ref{fig:New transformant}).

```{r New transformant, out.width = "80%", fig.align="center", echo=FALSE, fig.pos= "H", fig.cap="Pink lady shoot regeneration. \\newline{} \\textbf{A.} and \\textbf{B.} grafting and cotyledone like appearance"}
include_graphics("Transformant2 results.png") 
```

### Scale-up experiment  
At the scale up experiment for the purpose of statistical analysis of the methods found in the last two experiments. At this experiment about 800 seeds were planted from two apple varieties (Pink Lady and Granny Smith), out of which about half germinated. 400 plantlets were infected with 2 of the most promising plasmids (WOX11-STM and WUS2-STM) and 1 control plasmid (Empty). However, the plants were infected by Powdery mildew and Aphids diseases, hence no phenotype could be observed.

### High humidity experiment  
The objective of this experiment was to exclude any variables from the environment and maintain moisture for the cells that were exposed to air during the infection process. In the high humidity experiment about 500 seeds were planted in close containers with sterile soil. Next, these plants were infected with agrobacterium containing _WOX11_-_STM_ and _WUS2_-_STM_ plasmids, similarly to the Scale-up experiment. Unfortunately, with those conditions no shoot regeneration was observed.

### Poplar infections  
A similar experiment was performed on PopAT in which the infection was of soil grown plantlets were infected utilizing method C (Figure \ref{fig:Agro-infection methods}) applying the poplar constructs detailed in Table 5. Unfortunately, no phenotype as result of the treatment was observed.



### Identification of new transcription factors that are development regulators in poplar  
After several experiments in apples and poplar plants with DRs described in the litraturet and obtaining non-sufficient results. I tried and identify 'new' transcriptional factors that are expressed in the epical meristem. Tissue specific transcriptome data[@Grigoriev2012] was analyzed including shoot-tip transcriptome. For each tissue, I mapped the reads onto a transcriptome from _Populus trichocarpa_ and extracted the read count per accession (gene transcript). Next I filtered the accessions so that only those with that were there count was grater than at least an order of magnitude in the shoot tip compared to the other tissue examined (Root-tip, Bud, Bark, Xylem and leaf). After the filtration 85 accessions remained. I found that from the WUSHCEL-related gene family, WOX1 was by far the most expressed gene in the shoot-tip in comparison to other tissues (Figure \ref{fig:RNA-seq}) and was with the $8^{th}$ highest expression over all (Table \ref{tab:ShootSpecific}).

```{r RNA-seq, echo=FALSE, out.width = "80%", fig.align="center", echo=FALSE, fig.pos= "H", fig.cap="WOX1 expression detected by the RNA-seq analysis"}
WOX1_df <- read.csv("Tables/WOX1_df.csv")
RNA_plot <- barplot(WOX1_df$Expression, names = WOX1_df$Tissue, ylab = " WOX1 count", xlab = "Tissues")
```

```{r ShootSpecific, echo=FALSE, out.width = "100%", fig.align="center", echo=FALSE, fig.pos= "H"}
ShootSpecific_Over10 <- read.csv("Tables/ShootSpecific_Over10.csv")

ShootSpecific_Over10 <- ShootSpecific_Over10 %>% select(Accession, Description)

ShootSpecific_Over10$Description <- str_replace(ShootSpecific_Over10$Description, "PREDICTED: Populus trichocarpa ", "")


sep = c(rep('',nrow(ShootSpecific_Over10)))                             
kable(ShootSpecific_Over10[1:10,] , caption = "List of genes when sorted by the expression specificity to shoot tip (First 10)" ,booktabs = TRUE, linesep = sep) %>%
  kable_styling(latex_options = "HOLD_position") %>% row_spec(7, bold = TRUE)
```

Other genes found in this analysis may serve as a candidats for DR research, like MYB8. This protein has been described as HOS10 (high expression of osmotically responsive genes 10), a coordinating factor for responses to abiotic stress and for growth and development [@yanhui2006myb].


### Validation of the RNA-seq analysis with real-time quantitive PCR (RT-qPCR)
The RT-qPCR is a well established and highly sensitive method for the assessment of gene specific expression and hence can help validate the results of RNA-seq analysis. In most RT-qPCR experiments, the samples are from the same tissue which was exposed to different conditions and hence there are a few house-kipping gene that are commonly used for internal normalization. I performed, RT-qPCR on _WOX1_ for compering its expression in different tissues (RT-qPCR tissues). At first, I tried to use one of the common house-keeping gene, Actin-7, but, it exhibited a high variation in its expression between the tissues and can not be used as a reliable internal normalizer. Therefore, I returned to the RNA-seq data to look for other normalization genes. I chose the most unchanged genes across all tissues that were compared (RT-qPCR tissues). Excluding uncharacterized and mitochondrial genes the most uniformly expressed genes were Serine/threonine-protein phosphatase _PP2A-4_ catalytic subunit and E3 ubiquitin ligase. Hence, I used these two for normalization in the _WOX1_ RT-qPCR tests. In each test I compared the variation in the expression of the normalizing gene among all tissues calculated by the coefficient of variance (CV). In other words, I examined how different was the expression of each gene in the different tissues, as the CV get smaller, the expression is more uniform. Even though the RNA-seq analysis showed little variability in the expression of those genes in the different tissues, the RT-qPCR results showed otherwise (Figures \ref{fig:qPCR_normalizers}).


```{r qPCR_normalizers, out.width = "100%", fig.align="center", echo=FALSE, message=FALSE, fig.show="hold" ,fig.pos= "H", fig.cap="Expression of internal normalizers gene analyzed by RT-qPCR. The numbers below are the Coefficient of variation (CV) values and it represents a score for the amount in which the normalizer gene is expressed consistently between the different tissues."}

Normalizers_only <- read.csv("Tables/Normalizers_qPCR.csv")

AllNormalizers <- Normalizers_only %>% group_by(Target) %>% dplyr::mutate(CV = sd(Expression) / mean(Expression)) %>% dplyr::arrange(CV)

ggplot(data = AllNormalizers, aes(x = reorder(Target, CV), y = Expression)) + geom_boxplot() + theme_classic() + labs(x = "Internal normalizer genes", y = "Average expression") + geom_text(data = AllNormalizers, aes(x = reorder(Target, CV), y = -0.02, label=round(CV, 3)), size = 5)

```

For further analysis of _WOX1_, I analyzed the RT-qPCR results despite the normalizers inconsistency (Figures \ref{fig:qPCR}). The order in which the tissues appeared when sorted by the _WOX1_ gene expression remains consistent with the normalizers PP2A-4 and E3 against the RNA-seq analysis (Leaf > Xylem > Bark > RootTip > Bud), the only tissue that repositioned was the shoot tip, from third in E3 to last in PP2A-4. The fact that the results of position order of all tissues but one was correlated between seemingly unrelated normalizer genes, suggesting that those results are sound. Unfortunately, the tissue in question, Shoot-Tip wasn't correlated, hence it was impossible to deduce its true position, which was the information I was after.

```{r qPCR, echo=FALSE, out.width = "100%", fig.align="center", echo=FALSE, fig.show="hold" ,fig.pos= "H", fig.cap="WOX1 expression analyzed by the RT-qPCR with different internal normalizers."}
par(mfrow = c(1,3))

WOX1qPCR <- read.csv("Tables/WOX1qPCR.csv") %>% select(Normalizer, Tissue = BiologicalGroup, Expression)

qPCR_E3 <- WOX1qPCR %>% dplyr::filter(Normalizer == "E3") %>% dplyr::arrange(desc(Expression))
E3_plot = barplot(qPCR_E3$Expression, names = qPCR_E3$Tissue, ylim = c(0, 1), main = paste("E3 Ubiquitin ligase"), las = 2)

qPCR_actin <- WOX1qPCR %>% dplyr::filter(Normalizer == "Actin") %>% dplyr::arrange(desc(Expression))
Actin_plot = barplot(qPCR_actin$Expression, names = qPCR_actin$Tissue, ylim = c(0, 1), main = paste("Actin-7"), las = 2)

qPCR_PP2A_4 <- WOX1qPCR %>% dplyr::filter(Normalizer == "PP2A") %>% dplyr::arrange(desc(Expression))
PP2A_4_plot = barplot(qPCR_PP2A_4$Expression, names = qPCR_PP2A_4$Tissue, ylim = c(0, 1), main = paste("PP2A-4"), las = 2)

```

### _WOX1_ gene isolation from PopAT shoot tip
For the assembly of plasmids with the _WOX1_ gene as a DR for shoot regeneration, DNA was extracted and purified from PopAT’s shoot tip tissue. _WOX1_ was amplified using PCR and run on an agarose gel. The expected band based the genomic data was 1164 bp. However the band obtained was about ~1700 bp, ~500 bp longer than expected. I assumed that the gene might hold an intron, although it is not known to contain one. Consequently, I decided to isolate the gene from cDNA. After extraction and purification of RNA from the tissue, cDNA was prepared, and _WOX1_ was amplified. The band observed in the agarose gel was at the expected length of 1164 bp, proving that _WOX1_ in PopTA contains ~500 bp intron (Figures \ref{fig:wox1 isolation}). I used this product for the assembly of the _WOX1_ construct. 

```{r wox1 isolation, out.width = "100%", fig.align="center", echo=FALSE, fig.pos= "H", fig.cap="Isolation of the WOX1 gene from PopAT shoot tip"}
include_graphics("WOX1_isolation.png") 
```

# Discussion and Conclusions

Food security is one of the major challenges facing humanity in the 21st century [@FAO2011], especially in light of forecasts regarding climate change and population growth. Cereals are staple crops and account for as much as 44% of all agriculture land use for food production [(AREA USED FOR PRODUCTION (FAO 2017))](https://ourworldindata.org/grapher/global-agricultural-land-use-by-major-crop-type). Cereal cultivars are annual plant and as such are more prone to be effected by changes in irrigation and overall climate patterns [@Shiferaw2014]. In contrast, Trees, or specifically fruit trees in the context of food production, are more resilient to environmental changes manly due to their strong and deep roots. Consequently, trees have an important role in food security. Currently, fruit trees produce a small part of the world's overall food production. For trees to reach their full potential, crop improvement methods need to be more robust. Genome editing using CRISPR/Cas9 technology can contribute to this effort. For a method to be adopted, it needs to fulfill certain criteria such as, speed, ease of execution, cheap and minimal preliminary knowledge. The most common used method for crop improvement using a molecular genetic approach is by agrobacterium-mediated transformation. This method involves ex-plant in sterile environment on tissue culture medium. At present, for genome editing, the CRISPR/Cas9 system is ligated into agrobacterium T-DNA cassette. This method was a dramatic brake through in plant genetics but still it takes several months till soil-grown transformant plant is obtained and moreover it requires skilled personal for implementation and knowhow such as the specific growth medium and conditions per cultivar etc.

In-planta transformation, without involving cell culture, is an extremely desired goal. Zhang et al.(2017) developed a transformation method that mimics  tissue culture in-planta by creating a micro environment on an inner tissue section of the plant. The method was implemented on pomelo. a stem of a young plant was cut and the edge was inserted into a tube which was filled with liquid growth medium. From there on, the whole process was in line with other agrobacterium-mediated transformation procedures, co-culture with _Agrobacterium tumefaciens_ in the dark, adding of antibiotics and hormones (Figure \ref{fig:Pommelo}). several of the drawbacks in tissue culture methods are present here too, like the need for hormones and antibiotics.   

## DRs based tree transformation experiments 

Maher et al.(2020) developed a novel method for plant gene editing by introducing a defined combination of DRs into the cell as well as CRISPR/Cas9 and sgRNA, with the use of agrobacterium on soil-grown plants. As a result a de-novo shoot induction is obtained as well as editing in the gene of interest without using tissue culture. The experiment was performed on annual plants (tobacco, tomato) and one Perennial plant, grapevine. It should be noted that in most cases deformations occurred as a result of the over expression of the DRs.

In this work I attempted to implement this novel transformation method on trees, on poplar as a model system and on apple as one of the most economically important fruit trees. The potential of this method is sound and in trees it is even more striking when compared with traditional methods because all the reason discussed in the genetic editing bottleneck section.

The target in poplar was PDS because it is a well-established reporter gene for the verification of gene knockout (mutating the gene results in disruption of green pigmentation and the plant turns white). The target gene in apple was the S-RNase gene which regulates the self-incompatibility fertilization system (mutating the gene results in self-fertilization ability) [@DelDuca2019]. If this editing will succeed this will be a major step towards the construction of self-compatible apple cultivar which up to date do not exist. 

I examined three different strategies for introducing DNA into the plant using agrobacterium:  pressure injection to the leaf surface, injection to the leaf veins and injection to a stem cut. Our result was in favor of the latter, which is a similar method to the one described by Zhang et al.(2017) (Figure \ref{fig:Pommelo}).

Maher et al.(2020) aimed to show de-novo shoot induction as a proof of concept, hence their DNA introduction strategy was to surgically remove axillary buds and inject the agrobacterium to the cuts. Since my target gene in apples wasn't going to produce visual clues for gene editing, I was concerned for not beeing able to properly insure total removal of the axillary bud, and in order to avoid false positives for the regeneration as much as possible, I decided to not imitate their research on that regard. 

In comparison to Maher et al.(2020) that describe the work on _Nicotiana benthamiana_ and also mentioned work on tomato, potato and grape, we attempted to implement the method on poplar and apple trees. Another difference in comparison to their work appeared in the results. The new shoot formed on the top of the stem was not deformed (Figure \ref{fig:Deform_benthamiana}), possibly as a result of the nature of the growth, mimicking the natural shoot apical meristem, and the apical control with all its hormone flux involve in the process.

The infection methods were examined at first as a preliminary experiment only on one plant with each of the plasmids, resulted in regenerative plants when the plasmids harbored the DR combinations WOX11-STM and WUS-STM. The construct contained also the CRISPR system with a gRNA targeting the S-RNase gene however editing was not identified in the sequence of the S-RNase genes. For further verification a larger experiment was performed with the same plasmids (DRs experiment). 352 plants were infected. 44 of them with WOX11-STM, and 4 of them regenerated new shoot from the cut site. Other the the scale DRs experiment was different compared to the preliminary one in that the cut site was covered with parafilm for better moister maintenance [@Zhang2017].

In the scale-up experiment, both parafilm and aluminum foil was used to cover the stem cut for kipping dark and moist environment. The Dark environment is crucial in the first 48 hours for the agrobacterium infection. Unfortunately, as a result of disease spread, no phenotype was observed. 

In another experiment, to increase the possibility of infection success, the seeds were grown under aseptic conditions, on soil and were kept in a high humidity environment. However, this strategy did not help and no shoot regeneration was not formed.

Although some experiments I has technical problems I think that from the results that were obtained one may suggest that it is important to use plants at an exact growth stage that may be more susceptible to transformation with agrobacterium. Possibly seedlings are preferable than cuttings and probably an early stage of the seedling development – immediately after developing the first two leaves. Hence, I suggest that for future experiment to utilize this property of seedlings and all the other advantages of PopAT, it hould be wise to experiment on PopAT seedlings.

## _WOX1_ as a potential epical meristem transcription factor for DR induced transformation

To further attempts to increase the frequency of de-novo shoot formation, I pursue the identification of new transcription factors that are development regulators in trees and perhaps, under the right conditions, would act as a master regulators. Fortunately, in 2017 the [**DOE Joint Genome Institute**](https://genome.jgi.doe.gov/portal/PoptreSequencing_FD/PoptreSequencing_FD.info.html) [@Grigoriev2012] sequenced the _Populus tremula x alba_ INRA717-IB4 transcriptome.I analyzed the transcription profile per tissue, and identified genes that are strongly correlated to shoot development. In the analysis it appeared that out of the WUSCHEL-related homeobox (WOX) gene family, _WOX1_ had the strongest correlation to shoot formation. This finding corresponds to the Tvorogova et al.(2021), where _WOX1_ was found to regulate auxin response. This makes sense, since many of the genes whose expression is affected by WOX1 are involved in signaling pathways, transport, and synthesis of auxin. Furthermore, after narrowing the results to genes that are at least an order of magnitude shoot specific over all tissues, WOX1 came in &8^{th}& place out of 85. 

With strong corelation to our results of regeneration with the combination of genes _WOX11_-_STM_ and _WUS2_-_STM_, It has been shown that in the regulation of SAM, STM and WUS act in parallel, and they are necessary for the normal expression of each other [@Tvorogova2021]. This insight can also explain the non-deformed shoot formation I observe in contrast to Maher et al's.[@Maher2020] findings. 

_WOX11_ is known as a regulator of callus formation, development of adventitious roots, shoot regeneration and somatic embryogenesis [@Liu2018a].  

To validate our RNA-seq analysis, I extracted total RNA from similar tissues as the DOE's dataset (excluding callus, since it does not occur in natural growth). Then I run RT-qPCR analysis on the expression of _WOX1_ with various normalizer genes. Since the expression profile of the different tissues vary massively, it's difficult to find an appropriate normalizer gene with a similar expression level between the tissues. Despite that, I could still see the overall trend in which it does seemed that as the coefficient of variation (CV) got smaller, the results of the RT-qPCR become more similar to the RNA-seq results (Figure \ref{fig:qPCR}). When sorting the tissues by the expression of the gene _WOX1_ in the RT-qPCR results and in the RNA-seq analysis, the only tissue that shifted was shoot tip. The tissue is not well defined since is a combination of SAM, stem and leaves among other tissues and it is hard to isolate. Hence, the larg variability in the results can be explained by the resolution of the isolation of the tissue. 

In summary: the quest to achieve an in-planta transformation in trees is a huge challenge. In this work although I did not reach the final milestone of in-planta tree gene editing, still I hope that the work has closed part of the gap and will contribute to evoke de-novo shoot regeneration on a soil grown tree.  

Although our initial assumption was that it would be easier to transform poplar compared to apples, I didn’t take into account the influence of the different growth stages which I act upon. Perhaps with seedlings of PopAT I could get different results.

# Acknowledgements

# Supplementary information

[Analysis code link.](https://htmlpreview.github.io/?https://github.com/BenSiv/PopAT-expression-analysis/blob/main/PopAT_Expression.html)

```{r Bud_Excision, out.width = "100%", fig.align="center", echo=FALSE, fig.pos= "H", fig.cap="Surgical removal of the axillary bud and injection of vector for de-novo shoot induction (Maher et al. 2020)"}
include_graphics("Bud excision.png") 
```
```{r Deform_benthamiana, out.width = "100%", fig.align="center", echo=FALSE, fig.pos= "H", fig.cap="Abnormal shoot regeneration formation as a result of DR over-expression (Maher et al. 2020)"}
include_graphics("Deform benthamiana PDS.png") 
```
```{r Pommelo, out.width = "80%", fig.align="center", echo=FALSE, fig.pos= "H", fig.cap="Agrobacterium-mediated in-planta transformation for Citrus maxima. \\textbf{a.} Three to four week old C. maxima seedlings. \\textbf{b.} Decapitated C. maxima seedlings. \\textbf{c.} Agrobacterium infection. \\textbf{d.} Agrobacterium infected seedlings with wounds wrapped with Parafilm. \\textbf{e.} Dark incubation during co-culture. \\textbf{f.} sprouted bud from newly formed callus. \\textbf{g.} sprouted buds from xylem. \\textbf{h.} regenerated shoots four weeks after transformation. \\textbf{i.} regenerated shoots three months after transformation. (Zhang et al. 2017)"}
include_graphics("Pommelo transformation.png") 
```
****
## Protocols

### [Gel agarose for DNA electrophoresis](http://schepartzlab.yale.edu/intranet/protocols/AgaroseGelElectrophores.pdf)

Introduction:   

Preperation protocol of gel agarose for DNA electrophoresis.  

Materials:  

* Tray + Comb (With enough places for samples + ladder)
* LE multi-purpose agarose (Usually 1-2%)
* TAE * 1 Buffer (volume depend on Container)
* Ethidium bromide solution (1 drop for every 50 ml of gel)
* loading dye (1 µL loading dye per 5 µL sample) *Taq ready mix already contain dye.

Procedure:  

**Gel mixture, loading and running**  

1. Measure agarose by weight.
2. Dissolves in TAE buffer.
3. Microwave until fully dissolved.
4. Add ethidium bromide.
5. Pour into tray + comb.
6. Wait until the gel polymerizes.
7. Carefully remove the comb to expose the sample wells.
8. Place the gel into the tank.
9. Pour TAE buffer into the tank high enough to cover the gel.
10. Add loading dye (if necessary).
11. load DNA sample.
12. Connect the tank to the power supply.
13. Set the voltage at ~110 V for 30 minutes *The passage of current will produce bubbles at the electrodes.


### E.coli Heat-shock competent preperation [@Chang2017]  

Introduction:  

Preperation of E.coli competent for Heat shock transformation.  

Materials:  

* Liquid nitrogen
* ice
* LB with sutable antibiotics
* 20 mM $CaCl_2$ + 15% glycerol sterile

Procedure:  

**Pre-Culture**  

1. Pick single colony into a 4 ml LB (antibiotick if needed)
2. Grow at $37^oC$, 150 rpm overnight culture

**Culture**  

3. Add 0.5 ml of pre-culture to 50 ml LB, and Incubate untill $OD_{600}$ 0.5-1.
4. Chill culture on ice for 5-10 min.
5. transfer maximum 30 ml of the colture to a 50 ml tube.
6. Centrifuge for 10 min at 5000 rpm in $4^oC$.
7. Discard the supernated and let the tube to dry inverted for 1 min.
8. Add 0.5 ml ice cold 20 mM $CaCl_2$ + 15% glycerol.
9. Aliquot 100 $\mu l$ and freeze directly in liquid nitrogen.


### [E.coli heat-shock transformation](https://www.addgene.org/protocols/bacterial-transformation/) [@Chang2017]  

Introduction:   

Heat shock plasmid transformation to competent E.coli  

Materials:  

* Competent E.coli cells
* 50 $\mu l$ For each DNA construct / 100 $\mu l$ For ligation
* 50 ng of circular DNA
* Ice
* Water bath at $42^oC$
* 1.5 ml tube per sample (Eppendorf or similar)
* 1 ml of LB per sample (with no antibiotic added)
* LB + antibiotics plates 2 or 3 per sample for dilutions
* Drigalski spatula

Procedure:  

**Heat shock**  

1. Take competent E.coli cells from $-80^oC$ freezer. Use Top10 cells in most cases.
2. Turn on water bath to $42^oC$.
3. Put competent cells in a 1.5 ml tube (Eppendorf or similar). For transforming a DNA construct, use 50 $\mu l$ of competent cells. For transforming a ligation, use 100 $\mu l$ of competent cells. You may need more or less cells, depending how competent they are.
4. Keep tubes on ice.
5. Add 50 ng of circular DNA into E.coli cells. Incubate on ice for 20 min. to thaw competent cells.
6. Put tubes with DNA and E.coli into water bath at $42^oC$ for 1.5 min.
7. Put tubes back on ice for 5 minutes to reduce damage to the E.coli cells.
8. Add 1 ml of LB (with no antibiotic added). Incubate tubes for 1 hour at $37^oC$. (Can incubate tubes for 30 minutes, unless trying to grow DNA for ligation which is more sensitive. For ligation, leave tubes for 1 hour).
9. Spread about 100 $\mu l$ of the resulting culture on LB plates (with appropriate antibiotic added). Grow overnight.
10. Pick colonies about 12-16 hours later.


### CTAB protocol [@Porebski1997]  

Introduction:   

CTAB protocol for the isolating of DNA from plant tissues

Materials:  

* Liquid nitrogen
* Ice
* Pestles
* Stirer
* $65^oC$ Bath
* Centrifuge
* CTAB buffer
* 2% cetyl trimethylammonium bromide, 1% polyvinyl pyrrolidone,100 mM Tris-HCl, 1.4 M NaCl, 20 mM EDTA
* 2-Mercaptoethanol
* Chloroform octanol (1:24)
* Sodium acetate (NaOAc)
* 100% Ethanol
* 70% Ethanol

Procedure:  

1. Grind ~200 mg of plant tissue to a fine paste.
2. Add 700 $\mu l$ CTAB buffer pre-heated to $65^oC$ + 7$\mu l$ 2-Mercaptoethanol.
3. Vortex.
4. Incubation at $65^oC$ for 30 min (Vortex few times).
5. Chill on ice.
6. Add 700ul Chloroform octanol (1:24).
7. Vortex.
8. Centrifuge 5 min 14,000 rpm.
9. Transfer the upper aqueous phase only (contains the DNA) to a clean tube (~550-600 $\mu l$).
10. Add 400 $\mu l$ Chloroform octanol (1:24).
11. Centrifuge 5 min 14,000 rpm.
12. Transfer the upper aqueous phase only (contains the DNA) to a clean tube (~350 $\mu l$).
13. Add 35 $\mu l$ Sodium acetate (NaOAc), (X0.1 of sample volume).
14. Add 875 $\mu l$ 100% Ethanol, (X2.5 sample volume).
15. Incubate at $-80^oC$ for Hour/over night.
16. Centrifuge 10 min 10,000 rpm.
17. Remove the supernatant.
18. Add 1 ml 70% Ethanol.
19. Centrifuge 10 min 10,000 rpm.
20. Remove the supernatant.
21. Air dry the pellet, invert on paper towel.
22. Resuspend the DNA in 50 $\mu l$ sterile DNase free water.

### TRI Reagent® Protocol 

Introduction:   

TRI Reagent® solution (also sold as TRIzol) is a mixture of a mixture of guanidine thiocyanate and phenol in a monophase solution that is used for the isolation of DNA, RNA and protein from biological samples of human, animal, plant, yeast, bacteria, and virus. It inhibits RNase activity. TRI Reagent® is used to homogenize the biological sample from which RNA, DNA or proteins are extracted.

Materials:  

* Liquid nitrogen
* Pestles
* Centrifuge
* TRI Reagent®
* Chloroform
* 75% Ethanol


Procedure:  

1. Grind plant tissue to a fine paste with TRI Reagent (1 ml per 50–100 mg of tissue).
2. Incubation 5 minutes at room temperature.
3. Add 200 $\mu l$ chloroform per ml of TRI Reagent used.
4. Vortex.
5. Incubation 2–15 minutes at room temperature.
6. Centrifuge the resulting mixture at 12,000 g for 15 minutes at $2-8^oC$.
7. Transfer the aqueous phase to a fresh tube.
8. Add 0.5 ml of 2-propanol per ml of TRI Reagent used.
9. Mix gently.
10. Incubation for 5–10 minutes at room temperature.
11. Centrifuge at 12,000 g for 10 minutes at $2-8^oC$.
12. Discard the supernatant.
13. Add 1 ml of 75% ethanol per 1 ml of TRI Reagent used.
14. Vortex.
15. Centrifuge at 7,500 g for 5 minutes at $2-8^oC$.
16. Discard the supernatant.
17. Air dry the RNA pellet for 5–10 minutes.
18. Resuspend the RNA pellet with ~30 $\mu l$ of RNase free water.

# References
